diff of execution of:
echo 'export PATH=$PATH\':::::/hehe\'; echo $PATH; infile' | tr ';' '\n' | [shell]
--- expected	2024-09-05 15:28:56.898693914 +0200
+++ found	2024-09-05 15:28:56.914694303 +0200
@@ -1,6 +1,6 @@
 /home/bruno/.local/bin:/home/bruno/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/snap/bin:/home/bruno/.local/bin:/home/bruno/.local/bin:/home/bruno/.local/bin:/home/bruno/.local/bin:::::/hehe
 
 ======== stderr =======
-[progname]: infile: Permission denied
+[progname]: infile: command not found
 ======== return =======
 return value: 0
